### ISSUE TITLE
Is there an offical tensorflow version of soft nms?

### ISSUE THREAD

**Issue by zpge** [DocF/Soft-NMS#1]
Hi,
  Thanks for shaing your code. I want to ask is there an offical tensorflow version of soft nms. If I use a self-written cpu version of soft nms, like yours, will it greatly reduce the efficiency of the program? Thanks.

**Comment by vinitmuchhala** [comment]
Yes - https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/image/non_max_suppression_with_scores
if you set the 'soft_nms_sigma' variable to > 0 it will apply Gaussian Soft NMS on the candidate boxes and scores
Although, I should mention, I could not get it to work - different values of sigma did not affect the confidence scores
